scarb/src/ops/lint.rs

```diff
@@ -53,6 +56,8 @@ pub fn lint(opts: LintOptions, ws: &Workspace<'_>) -> Result<()> {
         },
     )?;
 
+    let absolute_path = opts.path.as_ref().and_then(|path| canonicalize(path).ok());
```

Comment:
I don't think we should silently ignore the error, when the provided path is invalid